core-api/src/main/java/com/sequenceiq/cloudbreak/client/CloudbreakClient.java

```diff
@@ -189,6 +190,10 @@ public UtilEndpoint utilEndpoint() {
         return refreshIfNeededAndGet(UtilEndpoint.class);
     }
 
+    public ImageCatalogV3Endpoint imageCatalogV3Endpoint() {
```

Comment:
I know it's not strictly part of your task but can you please add the other already implemented V3 endpoints here?

Reply:
At the end we have to check both CloudbreakClient and newway client to, so at this point it is not necessary i think

Reply:
yep thats also good